_components/language-selector/language-selector.md

```diff
@@ -40,7 +40,7 @@ title: Language selector
 type: component
 changelog:
   key: component-language-selector
-in_page_nav: false
```

Comment:
Similar to the header nav, I don't think the preview, code, guidance headers, etc. should have the same heading level as the language selector type under which they fall. This guidance should be nested at a lower heading level.